### PR TITLE
blog: flask-restx migration

### PR DESCRIPTION
**Pull request by nikromen**
fixes [#3231](https://github.com/fedora-copr/copr/issues/3231)

### CODE REVIEW

**Comment by praiskup** [comment]
missing images

**Comment by xsuchy** [comment]
Beside these images: +1

File: _posts/flask-restx-migration.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## Migration from Flask to Flask-RESTx for API documentation
+
+Hey everyone! We recently decided to switch COPR from Flask to Flask-RESTx, mainly to improve our API documentation since the documenting was non-existing.
```
Comment (FrostyX) [inline _posts/flask-restx-migration.md:3]: > decided to switch COPR from Flask to Flask-RESTx

Nitpick: We didn't switch the whole project, only the API.

File: _posts/flask-restx-migration.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - It ensures the data API handles is valid and correctly formatted.
+
+## What it looks like
+
+So, what does this all look like in practice? Let me give you a sneak peek.
+
+Instead of static, hard-to-update documentation, you get an interactive interface where users can do these things:
+
+- Explore Endpoints: Users can see all available API endpoints at a glance. Each endpoint is listed with a clear description of what it does.
+![alt text](endpoint-explore.png)
```
Comment (FrostyX) [inline _posts/flask-restx-migration.md:21]: Alt text is a placeholder :-)
Same for the images bellow.

**Review by FrostyX — COMMENTED** [review]
Minor notes, overall I like the post.

**Comment by praiskup** [comment]
Nit: I'd add more links, to flask-restx, swagger, pydantic/marshmallow (because those terms are not defined in this article, and defining them doesn't make sense - otoh the audience might be experienced enough, dunno).

Overall +1

**Comment by praiskup** [comment]
Thank you!

**Pull request merged**